packages/react-dom/src/events/ReactDOMEventReplaying.js

```diff
@@ -87,10 +90,8 @@ type PointerEvent = Event & {
   ...
 };
 
-import {IS_REPLAYED} from './EventSystemFlags';
-
 type QueuedReplayableEvent = {|
-  blockedOn: null | Container | SuspenseInstance,
+  blockedOn: typeof undefined | null | Container | SuspenseInstance,
```

Comment:
I don't love how the `undefined | null` propagated here and through all the call signatures. It's really easy to mess up in some check. Can we make `hasNullTarget` or something like this an explicit field in these objects and these functions? And determine it as early as possible after the `findInstanceBlockingEvent` call and thread it through explicitly.